### PR TITLE
Improve Azure startup time and fix command palette

### PR DESCRIPTION
**Pull request by kevcunnane**
- Improvement (but not full fix) for #4732 where Azure startup is very slow.
This speeds the startup by up to 5 seconds by
  - Changing from fixed 5 second initial wait time to a promise to check for accounts
  - Using the accounts changed notification to send a message when accounts are ready. This is usually what will "win out" since Azure seems to load before the Account Providers are set up.
- Remove right-click actions from the command palette.
- Rename Azure actions so it's clear what they are in the command palette.

### CODE REVIEW

**Comment by kevcunnane** [comment]
Note: build failure is due to coveralls upload not working. Please ignore, shouldn't block checkin.

File: extensions/azurecore/src/azureResource/tree/treeProvider.ts [hunk 1]
```diff
@@ -66,6 +75,23 @@ export class AzureResourceTreeProvider implements TreeDataProvider<TreeNode>, IA
...
+			// System has been initialized
+			this.setSystemInitialized();
+			this._onDidChangeTreeData.fire(undefined);
+		} catch (err) {
+			// Skip for now, we can assume that the accounts changed event will eventually notify instead
+			this.isSystemInitialized = false;
+		}
+	}
+
+	private setSystemInitialized() {
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/azureResource/tree/treeProvider.ts]: nit: void return type

File: extensions/azurecore/src/azureResource/tree/treeProvider.ts [hunk 2]
```diff
@@ -66,6 +75,23 @@ export class AzureResourceTreeProvider implements TreeDataProvider<TreeNode>, IA
...
+		this.isSystemInitialized = true;
+		this.loadingAccountsPromise = undefined;
+	}
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/azureResource/tree/treeProvider.ts]: Without having some sort of cancel isn't the promise still going to run and eventually return even if the onDidChangeAccounts is called first? What if in loadAccounts you checked after getAccounts finishes and see if isSystemInitialized is false and only set this.accounts to the returned accounts if that's the case?

Comment (kevcunnane) [reply extensions/azurecore/src/azureResource/tree/treeProvider.ts]: So if the call succeeds, the account manager is up & this is treated as loaded. This is still equivalent to the current behavior - on promise completion it'll cause getChildren to be called again, the accounts will be queried freshly & things "should" work. I'm going to won't fix this comment if that's OK.

**Pull request merged**